### PR TITLE
FWF-3834:[Feature] - Added Reusable Import Modal to MF

### PR DESCRIPTION
**Pull request by Josephalexantony-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3834
Issue Type:  FEATURE

# Changes
Reusable Import modal component in MF

### CODE REVIEW

File: forms-flow-components/src/components/SvgIcons/index.tsx [hunk 1]
```diff
@@ -635,3 +634,22 @@ export const TickIcon = ({ color = baseColor, ...props }) => (
...
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="16"
+    height="9"
+    viewBox="0 0 16 9"
+    fill="none"
+  >
+    <path
+      d="M1.5 1.24951L8.00058 7.7501L14.5006 1.25009"
+      stroke="#253DF4"
```
Comment (arun-s-aot) [inline forms-flow-components/src/components/SvgIcons/index.tsx:648]: pls add color as props with baseColor as default. then we can replace stroke="#253DF4" with stroke={color}

File: forms-flow-theme/scss/_modal.scss [hunk 2]
```diff
@@ -856,3 +856,71 @@
    }
  }
 }
+
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_modal.scss:859]: line 859 to 926 can be moved to fileUpload.scss as we have a separate file for import operations

File: forms-flow-theme/scss/_modal.scss [hunk 3]
```diff
@@ -856,3 +856,71 @@
...
+    padding: var(--spacer-200) 0 var(--spacer-200) 0;
+    align-items: center;
+    align-self: stretch;
+    border-bottom: 1px solid var(--ff-gray-100);
+
+    .import-layout-text,
+    .import-workflow-text {
+      width: 30%;
+      color: var(--gray-800);
+      font-size: 0.875rem;
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_modal.scss:914]: variable to add

File: forms-flow-theme/scss/_modal.scss [hunk 4]
```diff
@@ -856,3 +856,71 @@
...
+    align-items: center;
+    align-self: stretch;
+    border-bottom: 1px solid var(--ff-gray-100);
+
+    .file-item-header-text {
+      display: flex;
+      justify-content: flex-start;
+      width: 30%;
+      color: var(--ff-gray-800);
+      font-size: 0.875rem;
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_modal.scss:897]: replace with variabel

File: forms-flow-components/src/components/CustomComponents/ImportModal.tsx [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+        {selectedFile ? (
+          <>
+            <ProgressBar now={uploadProgress} />
+            <div className="upload-body">
+              <div className="upload-details">
+                <p className="upload-file-name">{selectedFile.name}</p>
+                <span className={`${!importLoader && !importError && !inprogress ? 'upload-status-success' :
+                  !importLoader && importError && !inprogress ? 'upload-status-error' : inprogress ? 'upload-status-progress' : ''}`}>
+                  {!importLoader && !importError && !inprogress ? (
+                    <Translation>{(t) => t("Upload Successful")}</Translation>
```
Comment (arun-s-aot) [inline forms-flow-components/src/components/CustomComponents/ImportModal.tsx:196]: please check whether transaltion can work without <translation> tag . using t() , it should work. pls check and if working, remove <translation> tag in entire FILE

File: forms-flow-components/src/components/CustomComponents/ImportModal.tsx [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+      <Modal.Header>
+        <Modal.Title>
+          <b><Translation>{(t) => t(headerText)}</Translation></b>
+        </Modal.Title>
+        <div className="d-flex align-items-center">
+          <CloseIcon width={16.5} height={16.5} onClick={() => { resetState(); closeModal(); }} />
+        </div>
+      </Modal.Header>
+      <Modal.Body className="p-5">
+        {selectedFile ? (
```
Comment (arun-s-aot) [inline forms-flow-components/src/components/CustomComponents/ImportModal.tsx:187]: as this is tsx file, better to reduce split the html to multiple sub fuctions for easy readability

**Review by arun-s-aot — APPROVED** [review]

**Review by Ajay-aot — APPROVED** [review]

**Pull request merged**